src/doc/style-guide/src/README.md

```diff
@@ -112,6 +112,11 @@ fn bar() {}
 fn baz() {}
 ```
 
+### Trailing whitespace
+
+Do not include trailing whitespace on the end of any line. This includes blank
+lines, comment lines, and code lines.
```

Comment:
I think it is worth calling out the case of multi-line string literals which contain trailing whitespace on a line, because

* handling them (whether by a formatter or manually) is not as simple as “delete the whitespace”, since that would change the meaning of the program, and
* there are multiple options for how to notate such spaces; for example, the last space only could be transformed to `\x20`, or all spaces could be, or `\n\` could be used in selected cases.